src/components/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Hey @okoroemeka, awesome job so far. Couple more things though:

I thought the PR that was merged this morning removed this file?

Why is it been reintroduced again?

Reply:
Resolved.